1-js/13-modules/02-import-export/article.md

```diff
@@ -63,7 +63,7 @@ export {sayHi, sayBye}; // a list of exported variables
 
 ...Or, technically we could put `export` above functions as well.
 
-## Import *
+## Import "*"
```

Comment:
disagree. all footnotes are signified with `(*)`, and this diverges from typical JS syntax